api/libproject/devstart.rb

```diff
@@ -198,6 +198,9 @@ def dev_up()
     ./gradlew loadConfig -Pconfig_key=featuredWorkspaces -Pconfig_file=config/featured_workspaces_local.json
   }
 
+  common.status "Loading Data Dictionary..."
+  common.run_inline %W{docker-compose run api-scripts ./gradlew loadDataDictionary -PappArgs=false}
```

Comment:
It looks like we're adding data dictionary to the deploy process? Is this part of the ticket or something extra?

Reply:
I realized that this is part of 
`Revert "Remove data dictionary tasks from dev scripts. (#2693)"` which reverts reverted code.

Reply:
👍 I'm reverting the commit that removed it from our dev-up and deploy tasks. It should be there IF it wasn't buggy.